cypher-shell/src/main/java/org/neo4j/shell/prettyprint/OutputFormatter.java

```diff
@@ -41,18 +46,41 @@
                 return relationshipAsString(value.asRelationship());
             case PATH:
                 return pathAsString(value.asPath());
+            case POINT:
+                return pointAsString(value.asPoint());
+            case DURATION:
             case ANY:
             case BOOLEAN:
+            case BYTES:
             case STRING:
             case NUMBER:
             case INTEGER:
             case FLOAT:
+            case DATE:
+            case TIME:
+            case DATE_TIME:
+            case LOCAL_TIME:
+            case LOCAL_DATE_TIME:
             case NULL:
             default:
                 return value.toString();
         }
     }
 
+    @Nonnull
+    default String pointAsString(Point point) {
+        StringBuilder stringBuilder = new StringBuilder("point({");
+        stringBuilder.append("srid:" + point.srid() + ",");
```

Comment:
I think string concatenation should not be used as an argument to `StringBuilder#append()`. It is better to either replace whole `StringBuilder` with `+` or use chained `#append()` calls.